system/ee/EllisLab/Addons/forum/mod.forum_core.php

```diff
@@ -5524,7 +5524,7 @@ public function submit_post()
 	 	// Is the IP or User Agent unavalable?
 		if (ee()->config->item('require_ip_for_posting') == 'y')
 		{
-			if (ee()->input->ip_address() == '0.0.0.0' OR ee()->session->userdata['user_agent'] == "")
+			if (ee()->input->ip_address() == EE_Input::DEFAULT_IP_ADDRESS OR ee()->session->userdata['user_agent'] == "")
```

Comment:
There's no `use EE_Input` statement for this file, is this an oversight or is there a different way it's invoked?

Reply:
Probably oversight